app/scripts/models/auth_brokers/base.js

```diff
@@ -75,7 +75,7 @@ define(function (require, exports, module) {
         success: t('Secondary email verified successfully')
       }),
       afterCompleteSignIn: new NavigateBehavior('signin_verified'),
-      afterCompleteSignInTokenCode: new NavigateBehavior('settings'),
+      afterCompleteSignInWithCode: new NavigateBehavior('settings'),
```

Comment:
I made this more generic since this is being reused for token codes and totp codes.